src/FileSystemUtilities.js

```diff
@@ -63,6 +64,12 @@ export default class FileSystemUtilities {
       }
       type = "file";
     }
+    if (pathIsAbsolute(src) && process.platform !== "win32") {
+      if (type === "junction") {
+        type = "dir";
+      }
+      src = relative(dirname(dest), src);
```

Comment:
So this is taking a path that was _already_ absolute and making it relative?

Where are we making it absolute?  Can we just skip that part in non-windows environments?

My worry here is we might take a dep that was _explicitly_ absolute and make it relative.